src/index.html

```diff
@@ -7,8 +7,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="logo" src="./images/logo.png" alt="logo" >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#home"  class="nav__link is-active" id="home">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#home" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#home" class="nav__link">Смартфоны и телефоны</a>
```

Comment:
Set different anchor links
```suggestion
            <a href="#apple"  class="nav__link is-active" id="home">apple</a>
          </li>
          <li class="nav__item">
            <a href="#samsung" class="nav__link">samsung</a>
          </li>
          <li class="nav__item">
            <a href="#phones" class="nav__link">Смартфоны и телефоны</a>
```